storage/blockdevice/tests/UNITTESTS/SFDP/test_sfdp.cpp

```diff
@@ -18,38 +18,126 @@
 #include "gmock/gmock.h"
 #include "blockdevice/internal/SFDP.h"
 
+using ::testing::_;
+using ::testing::MockFunction;
+using ::testing::Return;
+
+/**
+ * The Sector Map Parameter Table of Cypress S25FS512S:
+ * https://www.cypress.com/file/216376/download Table 71.
+ */
+static const mbed::bd_addr_t sector_map_start_addr = 0xD81000;
+static const uint8_t sector_map_multiple_descriptors[] = {
```

Comment:
It looks like this data is unused. Maybe we should instead add it with the commit that needs it.

Reply:
Done